### PR TITLE
feat: pulisher sh:node :AgentShape ->sh:class foaf:Agent

### PR DESCRIPTION
**Pull request by xiaofengleo**
pulisher sh:node :AgentShape ->sh:class foaf:Agent ;
addressing github issue 101 https://github.com/Health-RI/health-ri-metadata/issues/101

### CODE REVIEW

File: Formalisation(shacl)/Core/PiecesShape/DataService.ttl
Comment (kburger) [inline Formalisation(shacl)/Core/PiecesShape/DataService.ttl:1]: This file does not belong in this pull request. Please remove.

File: Formalisation(shacl)/Core/PiecesShape/DatasetSeries.ttl [hunk 1]
```diff
@@ -74,8 +74,8 @@
   ],
   [
     sh:path dct:publisher ;
-    #sh:class foaf:Agent ;
-    sh:node :AgentShape ;
+    sh:class foaf:Agent ;
+    #sh:node :AgentShape ;
```
Comment (kburger) [inline Formalisation(shacl)/Core/PiecesShape/DatasetSeries.ttl:78]: What is the intent here? This requires an IRI to be entered by the user, but will not allow them to fill in the nested fields. What do we want the user to enter?

**Review by kburger — CHANGES_REQUESTED** [review]

File: Formalisation(shacl)/Core/PiecesShape/DataService.ttl
Comment (xiaofengleo) [reply Formalisation(shacl)/Core/PiecesShape/DataService.ttl:1]: removed.

**Comment by kburger** [comment]
I don't see any changes since last time this issue was discussed. The conclusion then was to align with the solution from `dcat:Dataset`, as seen at https://github.com/Health-RI/health-ri-metadata/blob/develop/Formalisation(shacl)/Core/PiecesShape/Dataset.ttl#L85-L90.

**Review by kburger — APPROVED** [review]
LGTM, but did not test it on an FDP yet.

**Pull request merged**